tck/src/main/java/ee/jakarta/tck/concurrent/framework/signaturetest/README.md

```diff
@@ -0,0 +1,83 @@
+
+This package is inherited from the Platform-TCK (https://github.com/eclipse-ee4j/jakartaee-tck/tree/master/src/com/sun/ts/tests/signaturetest)
+
+The latest signature files are updated in the concurrency TCK Repository 
+(concurrency-api//tck/src/main/resources/ee/jakarta/tck/spec/signature) 
+generated using sigtest-maven-plugin.jar(version 1.6).
+
+The signature tests are run using the sigtest-maven-plugin.jar and the framework available in this folder.
+
+## For TCK developers
+
+### The configuration
+
+The signature file will be generated by the Concurrency team using the `tck/pom.xml`. 
+The pom.xml has a `signature-generation` profile configured for this purpose.
+The plugin that generates the signature file has been copied below for reference:
+
+
+```xml
+<plugin>
+	<groupId>org.netbeans.tools</groupId>
+	<artifactId>sigtest-maven-plugin</artifactId>
+	<version>${sigtest.version}</version>
+	<executions>
+		<execution>
+			<id>createSigFile</id>
+			<goals>
+				<goal>generate</goal>
+			</goals>
+		</execution>
+	</executions>
+	<configuration>
+		<classes>${project.build.directory}/concurrency-api</classes>
+		<packages>
+			jakarta.enterprise.concurrent,
+			jakarta.enterprise.concurrent.spi,
+		</packages>
+		<attach>false</attach>
+		<sigfile>${project.build.directory}/jakarta.enterprise.concurrent.sig_${version}</sigfile>
+	</configuration>
+</plugin>
+```
+
+- The `classes` configuration points to a location where an earlier plugin has unpacked the api jar form which we are generating the signature file.
+- The `packages` configuration specifies the packages we want to generate signatures from. 
+- The `attach` configuration specifies that we do not want the plugin to use it's default file extension
+- The `sigfile` configurations specifies the signature file we want the plugin to generate
+
+### Generating the Signature File
+
+To generate the signature file go to the TCK project:
+
+```sh
+cd tck
+```
+
+Use Maven to build the TCK and run the custom profile to generate the signature file:
+
+```sh
+mvn install -Psignature-generation
+```
+
+The signature file will be generated in the `/target/` directory.
+
+The signature file name expected is `jakarta.enterprise.concurrent.sig_${version}`, where version is the api version form which the signature is generated.
+
+Copy the signature file to the TCK so that it can be used by the test project, and checked into version control.
+`src/main/resources/ee/jakarta/tck/concurrent/spec/signature`
+
+## For TCK users
+
+### Running signature tests
+
+Since the Concurrency TCK can only be run against Jakarta EE Servers we have simplified the running of signature tests.
+
+Signature tests will be automatically run as part of the TCK via the `ee.jakarta.tck.concurrent.spec.signature` project. 
+
+This project will deploy a servlet application to the Jakarta EE Server. 
+The servlet that will run the signature test using the signaturetest framework from the Platform TCK.
```

Comment:
"The servlet that will run the signature test..."
should be
"The servlet runs the signature test..."